immutable-collection-encoding-test/pom.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hubspot.immutables</groupId>
+    <artifactId>hubspot-immutables</artifactId>
+    <version>1.3-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>immutable-collection-encoding-test</artifactId>
+
+  <properties>
+    <basepom.check.skip-findbugs>true</basepom.check.skip-findbugs>
+  </properties>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>com.google.errorprone</groupId>
+        <artifactId>error_prone_annotations</artifactId>
```

Comment:
Is it possible to get rid of the dependency on error prone? We do this in the normal immutales via https://github.com/HubSpot/hubspot-immutables/blob/e4c88a2a71f72e589a2f08ccaa9e582eeb9cb2da/hubspot-style/src/main/resources/META-INF/extensions/org.immutables.inhibit-classpath#L1

Reply:
👍